### ISSUE TITLE
Categorical data: add a stacked bar chart

### ISSUE THREAD

**Issue by JakeWags** [visdesignlab/upset2#204]

**Comment by JakeWags** [comment]
^ I think this is meant for the elementView sidebar?

**Comment by alexsb** [comment]
No, it's for the set view, where the box / violin plots go for numerical data. We currently don't have a test dataset though.

**Comment by JakeWags** [comment]
This is essentially adding support for categorical attributes


For WHO: countries population is numerical attribute, continents would be categorical

Horizontal bar chart for the attribute, percentage stacked bar maybe give both options


Tooltip for the legend
